packages/yarnpkg-core/sources/Report.ts

```diff
@@ -64,6 +64,7 @@ export enum MessageName {
   CACHE_OUTSIDE_PROJECT = 54,
   IMMUTABLE_INSTALL = 55,
   IMMUTABLE_CACHE = 56,
+  INVALID_WORKSPACE = 57
```

Comment:
Let's make it a bit more generic so that we can use it for validating any package.json 🤔

```suggestion
  INVALID_MANIFEST = 57
```